src/datatypes/aktivitetTypes.ts

```diff
@@ -15,7 +15,7 @@ export type AktivitetType =
 export type AktivitetStatus = 'AVBRUTT' | 'FULLFORT' | 'GJENNOMFORES' | 'PLANLAGT' | 'BRUKER_ER_INTERESSERT';
 export type StillingsStatus = 'INGEN_VALGT' | 'SOKNAD_SENDT' | 'INNKALT_TIL_INTERVJU' | 'AVSLAG' | 'JOBBTILBUD';
 export enum TransaksjonsType {
```

Comment:
Brukes denne for returtypen fra backenden?

Reply:
Ja. Vi har ikke noe eksplisitt mapping mellom API modellen og den interne domenemodellen i aktivitetsplan, og veilarbvarsel publiserer vel heller ikke noe skjema-artifakt, så vi må bare huske å oppdatere begge steder ved endringer.